### PR TITLE
Added file.flush after writing "chunks"

### PR DESCRIPTION
**Pull request by mscolly**
After having had larger files being corrupted when writing to the esp8266, I found that whole chunks sometimes went missing. This I corrected (and perhaps slowed the file transfer too) by doing a file.flush after every single write.

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -115,6 +115,7 @@ var config = {
 		execute: function (filename) {
 			new SerialComms(port).on('ready', function (comms) {
 				new DeviceManager(comms).executeFile(filename)
+                    .then(console.log)
```
Comment (paulcuth) [inline bin/cli.js]: If you could remove this `console.log()`, I'll happily merge.

Cheers.

**Comment by mscolly** [comment]
OK, done

**Comment by paulcuth** [comment]
Cheers.

**Comment by paulcuth** [comment]
re: the `console.log()`, I think you might need to rebase back onto my master as that line does already exist (from your last PR) but with different indentation.

**Pull request merged**